src/frontend/Komponenter/Behandling/Simulering/SimuleringTabellWrapper.tsx

```diff
@@ -27,6 +27,11 @@ const TekstMedMargin = styled(BodyShortSmall)`
     margin-top: 1rem;
 `;
 
+const StyledAlert = styled(Alert)`
+    max-width: 60rem;
+    margin: 1rem 0 0.25rem 0;
+`;
+
```

Comment:
Burde `SimuleringsContainer` ha ansvar for å sette spacingen på underliggende komponenter? Typ slik:
`display: flex; gap: 1rem; (eller 2 rem)`? Da kan du fjerne den individuelle spacing på komponentene i tillegg til at spacingen ikke differ fra komponent til komponent.

Reply:
Skal prøve å få til det